### PR TITLE
XW-553 | fix failing tests

### PR DESCRIPTION
**Pull request by tomasznapieralski**
https://wikia-inc.atlassian.net/browse/XW-553

* [x] MercuryArticleTest_004 - add wait before navigating to category page
* [x] MercuryCommentsTest_004 - add wait before navigating to user page
* [x] MercuryVKWidgetTest_004 - investigate why sometimes widgets are not visible, but edit and publish content helps
* [x] MercuryNavigationSideTest_001 - reassign webelements

### CODE REVIEW

**Comment by WikiaTech** [comment]
SonarQube analysis reported 28 issues:
* MAJOR 11 major
* MINOR 16 minor
* INFO 1 info

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MAJOR [Either log or rethrow this exception.](https://github.com/Wikia/selenium-tests/blob/be228f7581bdb9829255bf0f884f655b7b9172cb/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/mercury/BasePageObject.java#L101) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1166)
* MAJOR [Either log or rethrow this exception.](https://github.com/Wikia/selenium-tests/blob/be228f7581bdb9829255bf0f884f655b7b9172cb/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/mercury/BasePageObject.java#L158) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1166)
* MINOR [Define a constant instead of duplicating this literal "openMercuryArticleByName" 4 times.](https://github.com/Wikia/selenium-tests/blob/be228f7581bdb9829255bf0f884f655b7b9172cb/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/mercury/BasePageObject.java#L35) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* INFO [Complete the task associated to this TODO comment.](https://github.com/Wikia/selenium-tests/blob/be228f7581bdb9829255bf0f884f655b7b9172cb/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/mercury/BasePageObject.java#L110) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1135)
* MINOR [Define a constant instead of duplicating this literal "Article" 4 times.](https://github.com/Wikia/selenium-tests/blob/be228f7581bdb9829255bf0f884f655b7b9172cb/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/mercury/BasePageObject.java#L35) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal " was opened" 6 times.](https://github.com/Wikia/selenium-tests/blob/be228f7581bdb9829255bf0f884f655b7b9172cb/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/mercury/BasePageObject.java#L35) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MAJOR [Replace these toUpperCase()/toLowerCase() and equals() calls with a single equalsIgnoreCase() call.](https://github.com/Wikia/selenium-tests/blob/be228f7581bdb9829255bf0f884f655b7b9172cb/src/test/java/com/wikia/webdriver/testcases/mercurytests/ArticlePageTests.java#L283) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1157)
* MINOR [Define a constant instead of duplicating this literal "is not correct" 8 times.](https://github.com/Wikia/selenium-tests/blob/be228f7581bdb9829255bf0f884f655b7b9172cb/src/test/java/com/wikia/webdriver/testcases/mercurytests/ArticlePageTests.java#L201) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "URL for question mark" 4 times.](https://github.com/Wikia/selenium-tests/blob/be228f7581bdb9829255bf0f884f655b7b9172cb/src/test/java/com/wikia/webdriver/testcases/mercurytests/ArticlePageTests.java#L209) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MAJOR [Replace these toUpperCase()/toLowerCase() and equals() calls with a single equalsIgnoreCase() call.](https://github.com/Wikia/selenium-tests/blob/be228f7581bdb9829255bf0f884f655b7b9172cb/src/test/java/com/wikia/webdriver/testcases/mercurytests/ArticlePageTests.java#L197) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1157)
* MAJOR [Replace these toUpperCase()/toLowerCase() and equals() calls with a single equalsIgnoreCase() call.](https://github.com/Wikia/selenium-tests/blob/be228f7581bdb9829255bf0f884f655b7b9172cb/src/test/java/com/wikia/webdriver/testcases/mercurytests/ArticlePageTests.java#L352) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1157)
* MAJOR [Either log or rethrow this exception.](https://github.com/Wikia/selenium-tests/blob/be228f7581bdb9829255bf0f884f655b7b9172cb/src/test/java/com/wikia/webdriver/testcases/mercurytests/ArticlePageTests.java#L178) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1166)
* MINOR [Define a constant instead of duplicating this literal "Article title for colon" 4 times.](https://github.com/Wikia/selenium-tests/blob/be228f7581bdb9829255bf0f884f655b7b9172cb/src/test/java/com/wikia/webdriver/testcases/mercurytests/ArticlePageTests.java#L199) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MAJOR [Replace these toUpperCase()/toLowerCase() and equals() calls with a single equalsIgnoreCase() call.](https://github.com/Wikia/selenium-tests/blob/be228f7581bdb9829255bf0f884f655b7b9172cb/src/test/java/com/wikia/webdriver/testcases/mercurytests/ArticlePageTests.java#L240) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1157)
* MAJOR [Replace these toUpperCase()/toLowerCase() and equals() calls with a single equalsIgnoreCase() call.](https://github.com/Wikia/selenium-tests/blob/be228f7581bdb9829255bf0f884f655b7b9172cb/src/test/java/com/wikia/webdriver/testcases/mercurytests/ArticlePageTests.java#L305) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1157)
* MAJOR [Replace these toUpperCase()/toLowerCase() and equals() calls with a single equalsIgnoreCase() call.](https://github.com/Wikia/selenium-tests/blob/be228f7581bdb9829255bf0f884f655b7b9172cb/src/test/java/com/wikia/webdriver/testcases/mercurytests/ArticlePageTests.java#L260) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1157)
* MINOR [Define a constant instead of duplicating this literal "is not visible" 6 times.](https://github.com/Wikia/selenium-tests/blob/be228f7581bdb9829255bf0f884f655b7b9172cb/src/test/java/com/wikia/webdriver/testcases/mercurytests/ArticlePageTests.java#L45) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MAJOR [Replace these toUpperCase()/toLowerCase() and equals() calls with a single equalsIgnoreCase() call.](https://github.com/Wikia/selenium-tests/blob/be228f7581bdb9829255bf0f884f655b7b9172cb/src/test/java/com/wikia/webdriver/testcases/mercurytests/ArticlePageTests.java#L216) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1157)
* MINOR [Define a constant instead of duplicating this literal "is visible" 6 times.](https://github.com/Wikia/selenium-tests/blob/be228f7581bdb9829255bf0f884f655b7b9172cb/src/test/java/com/wikia/webdriver/testcases/mercurytests/ArticlePageTests.java#L44) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "is correct" 8 times.](https://github.com/Wikia/selenium-tests/blob/be228f7581bdb9829255bf0f884f655b7b9172cb/src/test/java/com/wikia/webdriver/testcases/mercurytests/ArticlePageTests.java#L200) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "Article title for question mark" 4 times.](https://github.com/Wikia/selenium-tests/blob/be228f7581bdb9829255bf0f884f655b7b9172cb/src/test/java/com/wikia/webdriver/testcases/mercurytests/ArticlePageTests.java#L218) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "is encoded" 8 times.](https://github.com/Wikia/selenium-tests/blob/be228f7581bdb9829255bf0f884f655b7b9172cb/src/test/java/com/wikia/webdriver/testcases/mercurytests/ArticlePageTests.java#L192) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "URL for colon" 4 times.](https://github.com/Wikia/selenium-tests/blob/be228f7581bdb9829255bf0f884f655b7b9172cb/src/test/java/com/wikia/webdriver/testcases/mercurytests/ArticlePageTests.java#L191) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "is not encoded" 8 times.](https://github.com/Wikia/selenium-tests/blob/be228f7581bdb9829255bf0f884f655b7b9172cb/src/test/java/com/wikia/webdriver/testcases/mercurytests/ArticlePageTests.java#L193) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MAJOR [Replace these toUpperCase()/toLowerCase() and equals() calls with a single equalsIgnoreCase() call.](https://github.com/Wikia/selenium-tests/blob/be228f7581bdb9829255bf0f884f655b7b9172cb/src/test/java/com/wikia/webdriver/testcases/mercurytests/ArticlePageTests.java#L329) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1157)
* MINOR [Define a constant instead of duplicating this literal "is not displayed" 7 times.](https://github.com/Wikia/selenium-tests/blob/be228f7581bdb9829255bf0f884f655b7b9172cb/src/test/java/com/wikia/webdriver/testcases/mercurytests/CommentsTests.java#L75) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "is correct" 3 times.](https://github.com/Wikia/selenium-tests/blob/be228f7581bdb9829255bf0f884f655b7b9172cb/src/test/java/com/wikia/webdriver/testcases/mercurytests/CommentsTests.java#L66) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "is displayed" 9 times.](https://github.com/Wikia/selenium-tests/blob/be228f7581bdb9829255bf0f884f655b7b9172cb/src/test/java/com/wikia/webdriver/testcases/mercurytests/CommentsTests.java#L74) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)

File: src/test/java/com/wikia/webdriver/testcases/mercurytests/widgettests/VKTests.java [hunk 1]
```diff
@@ -27,6 +29,10 @@
   @BeforeMethod(alwaysRun = true)
   public void prepareTest() {
     driver.manage().timeouts().pageLoadTimeout(100, TimeUnit.SECONDS);
+
+    //@TODO XW-314 - Login is necessary to bypass cache
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/mercurytests/widgettests/VKTests.java]: INFO Complete the task associated to this TODO comment. [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1135)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue:
* INFO 1 info

Watch the comments in this conversation to review them.

File: src/test/java/com/wikia/webdriver/testcases/mercurytests/widgettests/VKTests.java (hunk 1 above)
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/mercurytests/widgettests/VKTests.java]: INFO Complete the task associated to this TODO comment. [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1135)

File: src/test/java/com/wikia/webdriver/testcases/mercurytests/NavigationSideTests.java [hunk 2]
```diff
@@ -16,6 +16,7 @@
 /**
  * @ownership Content X-Wing Wikia
  */
+@Test(groups = {"MercuryNavigationSideTests", "Mercury"})
 public class NavigationSideTests extends NewTestTemplate {
 
   private final static String SEARCH_PASS = "Gallery";
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/mercurytests/NavigationSideTests.java]: MINOR Reorder the modifiers to comply with the Java Language Specification. [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AModifiersOrderCheck)

File: src/test/java/com/wikia/webdriver/testcases/mercurytests/NavigationSideTests.java [hunk 3]
```diff
@@ -245,11 +245,10 @@ public void MercuryNavigationSideTest_003_ValidSuggestionRedirect() {
   // NST04
   // TODO: check article titles instead of url, check that receive at least 3 different articles in 10 attempts
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/mercurytests/NavigationSideTests.java]: INFO Complete the task associated to this TODO comment. [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1135)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 39 issues:
* MAJOR 11 major
* MINOR 25 minor
* INFO 3 info

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* INFO [Complete the task associated to this TODO comment.](https://github.com/Wikia/selenium-tests/blob/825272e14d63d2e0b25d52c5e0528d4f2767f8c1/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/mercury/BasePageObject.java#L110) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1135)
* MINOR [Define a constant instead of duplicating this literal " was opened" 6 times.](https://github.com/Wikia/selenium-tests/blob/825272e14d63d2e0b25d52c5e0528d4f2767f8c1/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/mercury/BasePageObject.java#L35) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MAJOR [Either log or rethrow this exception.](https://github.com/Wikia/selenium-tests/blob/825272e14d63d2e0b25d52c5e0528d4f2767f8c1/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/mercury/BasePageObject.java#L158) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1166)
* MINOR [Define a constant instead of duplicating this literal "openMercuryArticleByName" 4 times.](https://github.com/Wikia/selenium-tests/blob/825272e14d63d2e0b25d52c5e0528d4f2767f8c1/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/mercury/BasePageObject.java#L35) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MAJOR [Either log or rethrow this exception.](https://github.com/Wikia/selenium-tests/blob/825272e14d63d2e0b25d52c5e0528d4f2767f8c1/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/mercury/BasePageObject.java#L101) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1166)
* MINOR [Define a constant instead of duplicating this literal "Article" 4 times.](https://github.com/Wikia/selenium-tests/blob/825272e14d63d2e0b25d52c5e0528d4f2767f8c1/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/mercury/BasePageObject.java#L35) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MAJOR [Replace these toUpperCase()/toLowerCase() and equals() calls with a single equalsIgnoreCase() call.](https://github.com/Wikia/selenium-tests/blob/825272e14d63d2e0b25d52c5e0528d4f2767f8c1/src/test/java/com/wikia/webdriver/testcases/mercurytests/ArticlePageTests.java#L283) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1157)
* MINOR [Define a constant instead of duplicating this literal "is correct" 8 times.](https://github.com/Wikia/selenium-tests/blob/825272e14d63d2e0b25d52c5e0528d4f2767f8c1/src/test/java/com/wikia/webdriver/testcases/mercurytests/ArticlePageTests.java#L200) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MAJOR [Replace these toUpperCase()/toLowerCase() and equals() calls with a single equalsIgnoreCase() call.](https://github.com/Wikia/selenium-tests/blob/825272e14d63d2e0b25d52c5e0528d4f2767f8c1/src/test/java/com/wikia/webdriver/testcases/mercurytests/ArticlePageTests.java#L352) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1157)
* MAJOR [Replace these toUpperCase()/toLowerCase() and equals() calls with a single equalsIgnoreCase() call.](https://github.com/Wikia/selenium-tests/blob/825272e14d63d2e0b25d52c5e0528d4f2767f8c1/src/test/java/com/wikia/webdriver/testcases/mercurytests/ArticlePageTests.java#L216) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1157)
* MINOR [Define a constant instead of duplicating this literal "is visible" 6 times.](https://github.com/Wikia/selenium-tests/blob/825272e14d63d2e0b25d52c5e0528d4f2767f8c1/src/test/java/com/wikia/webdriver/testcases/mercurytests/ArticlePageTests.java#L44) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "is not visible" 6 times.](https://github.com/Wikia/selenium-tests/blob/825272e14d63d2e0b25d52c5e0528d4f2767f8c1/src/test/java/com/wikia/webdriver/testcases/mercurytests/ArticlePageTests.java#L45) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MAJOR [Replace these toUpperCase()/toLowerCase() and equals() calls with a single equalsIgnoreCase() call.](https://github.com/Wikia/selenium-tests/blob/825272e14d63d2e0b25d52c5e0528d4f2767f8c1/src/test/java/com/wikia/webdriver/testcases/mercurytests/ArticlePageTests.java#L305) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1157)
* MINOR [Define a constant instead of duplicating this literal "is not encoded" 8 times.](https://github.com/Wikia/selenium-tests/blob/825272e14d63d2e0b25d52c5e0528d4f2767f8c1/src/test/java/com/wikia/webdriver/testcases/mercurytests/ArticlePageTests.java#L193) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "is encoded" 8 times.](https://github.com/Wikia/selenium-tests/blob/825272e14d63d2e0b25d52c5e0528d4f2767f8c1/src/test/java/com/wikia/webdriver/testcases/mercurytests/ArticlePageTests.java#L192) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "is not correct" 8 times.](https://github.com/Wikia/selenium-tests/blob/825272e14d63d2e0b25d52c5e0528d4f2767f8c1/src/test/java/com/wikia/webdriver/testcases/mercurytests/ArticlePageTests.java#L201) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MAJOR [Replace these toUpperCase()/toLowerCase() and equals() calls with a single equalsIgnoreCase() call.](https://github.com/Wikia/selenium-tests/blob/825272e14d63d2e0b25d52c5e0528d4f2767f8c1/src/test/java/com/wikia/webdriver/testcases/mercurytests/ArticlePageTests.java#L197) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1157)
* MAJOR [Either log or rethrow this exception.](https://github.com/Wikia/selenium-tests/blob/825272e14d63d2e0b25d52c5e0528d4f2767f8c1/src/test/java/com/wikia/webdriver/testcases/mercurytests/ArticlePageTests.java#L178) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1166)
* MINOR [Define a constant instead of duplicating this literal "URL for question mark" 4 times.](https://github.com/Wikia/selenium-tests/blob/825272e14d63d2e0b25d52c5e0528d4f2767f8c1/src/test/java/com/wikia/webdriver/testcases/mercurytests/ArticlePageTests.java#L209) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "Article title for colon" 4 times.](https://github.com/Wikia/selenium-tests/blob/825272e14d63d2e0b25d52c5e0528d4f2767f8c1/src/test/java/com/wikia/webdriver/testcases/mercurytests/ArticlePageTests.java#L199) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MAJOR [Replace these toUpperCase()/toLowerCase() and equals() calls with a single equalsIgnoreCase() call.](https://github.com/Wikia/selenium-tests/blob/825272e14d63d2e0b25d52c5e0528d4f2767f8c1/src/test/java/com/wikia/webdriver/testcases/mercurytests/ArticlePageTests.java#L240) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1157)
* MAJOR [Replace these toUpperCase()/toLowerCase() and equals() calls with a single equalsIgnoreCase() call.](https://github.com/Wikia/selenium-tests/blob/825272e14d63d2e0b25d52c5e0528d4f2767f8c1/src/test/java/com/wikia/webdriver/testcases/mercurytests/ArticlePageTests.java#L329) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1157)
* MAJOR [Replace these toUpperCase()/toLowerCase() and equals() calls with a single equalsIgnoreCase() call.](https://github.com/Wikia/selenium-tests/blob/825272e14d63d2e0b25d52c5e0528d4f2767f8c1/src/test/java/com/wikia/webdriver/testcases/mercurytests/ArticlePageTests.java#L260) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1157)
* MINOR [Define a constant instead of duplicating this literal "URL for colon" 4 times.](https://github.com/Wikia/selenium-tests/blob/825272e14d63d2e0b25d52c5e0528d4f2767f8c1/src/test/java/com/wikia/webdriver/testcases/mercurytests/ArticlePageTests.java#L191) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "Article title for question mark" 4 times.](https://github.com/Wikia/selenium-tests/blob/825272e14d63d2e0b25d52c5e0528d4f2767f8c1/src/test/java/com/wikia/webdriver/testcases/mercurytests/ArticlePageTests.java#L218) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "is not displayed" 7 times.](https://github.com/Wikia/selenium-tests/blob/825272e14d63d2e0b25d52c5e0528d4f2767f8c1/src/test/java/com/wikia/webdriver/testcases/mercurytests/CommentsTests.java#L75) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "is displayed" 9 times.](https://github.com/Wikia/selenium-tests/blob/825272e14d63d2e0b25d52c5e0528d4f2767f8c1/src/test/java/com/wikia/webdriver/testcases/mercurytests/CommentsTests.java#L74) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "is correct" 3 times.](https://github.com/Wikia/selenium-tests/blob/825272e14d63d2e0b25d52c5e0528d4f2767f8c1/src/test/java/com/wikia/webdriver/testcases/mercurytests/CommentsTests.java#L66) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Reorder the modifiers to comply with the Java Language Specification.](https://github.com/Wikia/selenium-tests/blob/825272e14d63d2e0b25d52c5e0528d4f2767f8c1/src/test/java/com/wikia/webdriver/testcases/mercurytests/NavigationSideTests.java#L23) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AModifiersOrderCheck)
* MINOR [Define a constant instead of duplicating this literal "is hidden" 8 times.](https://github.com/Wikia/selenium-tests/blob/825272e14d63d2e0b25d52c5e0528d4f2767f8c1/src/test/java/com/wikia/webdriver/testcases/mercurytests/NavigationSideTests.java#L44) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "Navigation menu" 3 times.](https://github.com/Wikia/selenium-tests/blob/825272e14d63d2e0b25d52c5e0528d4f2767f8c1/src/test/java/com/wikia/webdriver/testcases/mercurytests/NavigationSideTests.java#L43) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "is visible" 5 times.](https://github.com/Wikia/selenium-tests/blob/825272e14d63d2e0b25d52c5e0528d4f2767f8c1/src/test/java/com/wikia/webdriver/testcases/mercurytests/NavigationSideTests.java#L57) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "works" 3 times.](https://github.com/Wikia/selenium-tests/blob/825272e14d63d2e0b25d52c5e0528d4f2767f8c1/src/test/java/com/wikia/webdriver/testcases/mercurytests/NavigationSideTests.java#L126) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "is displayed" 3 times.](https://github.com/Wikia/selenium-tests/blob/825272e14d63d2e0b25d52c5e0528d4f2767f8c1/src/test/java/com/wikia/webdriver/testcases/mercurytests/NavigationSideTests.java#L91) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "Search suggestions" 3 times.](https://github.com/Wikia/selenium-tests/blob/825272e14d63d2e0b25d52c5e0528d4f2767f8c1/src/test/java/com/wikia/webdriver/testcases/mercurytests/NavigationSideTests.java#L176) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "Back link" 3 times.](https://github.com/Wikia/selenium-tests/blob/825272e14d63d2e0b25d52c5e0528d4f2767f8c1/src/test/java/com/wikia/webdriver/testcases/mercurytests/NavigationSideTests.java#L83) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)

**Comment by tomasznapieralski** [comment]
tests are passing locally and on Jenkins

**Pull request merged**